.github/workflows/test.yml

```diff
@@ -154,7 +167,7 @@ jobs:
 
       - name: Grab PHPUnit version
         id: phpunit_version
-        run: echo "VERSION=$(vendor/bin/phpunit --version | grep --only-matching --max-count=1 --extended-regexp '\b[0-9]+\.[0-9]+')" >> $GITHUB_OUTPUT
```

Comment:
windows doesn't like `\b`

Reply:
Some things just make me sad... :woman_facepalming:

I have a feeling you may still need to quote the `vendor/bin/phpunit` part of the command.

Reply:
all tests are green.. or did I missing something?

I guess because of the changed matrix you need to update the branch protection rules in the github repo settings

Reply:
pushed another fix.